packages/ndla-ui/src/Resource/resourceComponents.tsx

```diff
@@ -91,13 +97,18 @@ const TagCounterWrapper = styled.p`
 interface TagListProps {
   tags?: string[];
 }
-
 export const TagList = ({ tags }: TagListProps) => {
   if (!tags) return null;
   return (
     <StyledTagList>
       {tags.map((tag, i) => (
-        <StyledTagListElement key={`tag-${i}`}>{tag}</StyledTagListElement>
+        <StyledTagListElement key={`tag-${i}`}>
+          <StyledSafeLink
+            onClick={(e: MouseEvent<HTMLAnchorElement | HTMLElement>) => e.stopPropagation()}
+            to={`/minndla/tags/${tag}`}>
```

Comment:
Url bør vel sendes inn som en parameter i og med at komponenten er en generisk Resource som kan brukes andre plasser. Dette knytter den for tett til MinNDLA.

Reply:
Fikset det nå, - ser det greit ut?